core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/loadbalancer/AwsLoadBalancerResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.loadbalancer;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.NotNull;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class AwsLoadBalancerResponse implements Serializable {
+
+    @ApiModelProperty
```

Comment:
I think should add some value to this annotation, same for `AwsTargetGroupResponse`